automatic/qbittorrent/tools/VERIFICATION.txt

```diff
@@ -0,0 +1,11 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The installer have been downloaded from the alternative sourceforge mirror listed on <http://www.qbittorrent.org/download.php>
+and can be verified like this:
+
+1. Go to <http://sourceforge.net/projects/qbittorrent/files/qbittorrent-win32/qbittorrent-3.3.7/qbittorrent_3.3.7_setup.exe/download>
+   and download the installer
+2. Use powershell function 'Get-FileHash' or the chocolatey utility 'checksum.exe'
+   to verify that the installer matches the sha256 checksum '49AE9A0ADFC3272BEC38822C528F732D9495B79A2A7CA934F8C6635237B15D07'
```

Comment:
no problem, is there anything else should be in the verification.txt file?
or any additional steps I need to include?